example/config

```diff
@@ -31,35 +32,22 @@ BR2_x86_64=y
 # BR2_xtensa is not set
 BR2_ARCH="x86_64"
 BR2_ENDIAN="LITTLE"
-BR2_GCC_TARGET_TUNE="generic"
-BR2_x86_generic=y
```

Comment:
Yeah, saw those. Should be fine.

On Mon, Mar 16, 2015 at 7:52 PM, Andy Shinn <notifications@github.com>
wrote:

> In example/config
> <https://github.com/progrium/rootbuilder/pull/4#discussion_r26541670>:
>
> > @@ -31,35 +32,22 @@ BR2_x86_64=y
> >  # BR2_xtensa is not set
> >  BR2_ARCH="x86_64"
> >  BR2_ENDIAN="LITTLE"
> > -BR2_GCC_TARGET_TUNE="generic"
> > -BR2_x86_generic=y
>
> BR2_x86_generic was a deprecated option and BR2_GCC_TARGET_TUNE was
> changed to be nocona (there is no more generic). I think these were the
> two most glaring changes the *could* possibly affect a BusyBox rebuild.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/progrium/rootbuilder/pull/4/files#r26541670>.
>



-- 
Jeff Lindsay
http://progrium.com